pyproject.toml

```diff
@@ -0,0 +1,39 @@
+[build-system]
+requires = ["setuptools >= 61.0"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "x4i"
+version = "1.0.4"
```

Comment:
this is a duplicate of the version in the __init__.py. The toml format permits using an external version file, so you could use a single source for the version number.